collectives/routes/auth.py

```diff
@@ -102,6 +102,17 @@ def login():
 
     login_user(user, remember=form.remember_me.data)
 
+    # for user with a role which don't hve signed the confidentiality agreement
+    # We ask them to signed. This is non restrictive as long as user does note
+    # access a user profile
```

Comment:
Typos: 'have', 'to sign', 'does not'
Our reformulation:

We ask users with roles to sign the confidentiality agreement. Signature is compulsory to view user profiles.

Reply:
J'ai réutilisé ta formulation